bitnami/nginx/templates/deployment.yaml

```diff
@@ -234,6 +234,10 @@ spec:
           resources: {{- toYaml .Values.resources | nindent 12 }}
```

Comment:
you could add a checksum annotation of the configmap to the pod metadata for automatic reload of the pod when config changes